libs/chain-signatures/contract/readme.md

```diff
@@ -87,4 +87,8 @@ These functions require the caller to be a participant or candidate.
 Run tests with `cargo nextest run`. This includes the integration tests, for which it is necessary to compile the contract first:
 ```
 cargo build --release --target=wasm32-unknown-unknown
+wasm-opt -Oz -o mpc-contract.wasm target/wasm32-unknown-unknown/release/mpc_contract.wasm
+mv mpc-contract.wasm target/wasm32-unknown-unknown/release/mpc_contract.wasm
```

Comment:
Can you add this to the pytest fixture?

https://github.com/Near-One/mpc/blob/61c4d3ce6ae03ccd3b16f76e083f332f5dc3f3c4/pytest/tests/conftest.py#L42